src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String SEPARATOR = ",";
+    private static final int FIRST = 0;
+    private static final int SECOND = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] data = readFromFile(fromFileName);
+        String report = generateReport(data);
+        writeToFile(toFileName, report);
+    }
+
+    private int[] readFromFile(String fromFileName) {
+        int[] result = new int[] {FIRST,FIRST};
```

Comment:
```suggestion
        int[] result = new int[] {FIRST, FIRST};
```
don't forget to auto format your files before committing - Ctrl Alt L for windows